scripts/sct_download_data.py

```diff
@@ -14,47 +14,57 @@
 
 import os
 import sys
+import argparse
 
 from spinalcordtoolbox.download import install_data
-
-from msct_parser import Parser
+from spinalcordtoolbox.utils import Metavar, SmartFormatter, ActionCreateFolder
 import sct_utils as sct
 
 
 def get_parser(dict_url):
-    parser = Parser(__file__)
-    parser.usage.set_description('''Download binaries from the web.''')
-    parser.add_option(
-        name="-d",
-        type_value="multiple_choice",
-        description="Name of the dataset.",
-        mandatory=True,
-        example=sorted(dict_url))
-    parser.add_option(
-        name="-v",
-        type_value="multiple_choice",
-        description="Verbose. 0: nothing. 1: basic. 2: extended.",
-        mandatory=False,
-        default_value=1,
-        example=['0', '1', '2'])
-    parser.add_option(
-        name="-o",
-        type_value="folder_creation",
-        description=("Path where to save the downloaded data"
-         " (defaults to ./${dataset-name})."
-         " Directory will be created."
-         " Warning: existing data in the directory will be erased unless -k is provided."),
-        mandatory=False)
-    parser.add_option(
-        name="-h",
-        type_value=None,
-        description="Display this help",
-        mandatory=False)
-    parser.add_option(
-        name="-k",
-        type_value=None,
-        description="Keep existing data in destination directory",
-        mandatory=False)
+    parser = argparse.ArgumentParser(
+        description="Download binaries from the web.",
+        formatter_class=SmartFormatter,
+        add_help=None,
+        prog=os.path.basename(__file__).strip(".py")
+    )
+    mandatory = parser.add_argument_group("\nMANDATORY ARGUMENTS")
+    mandatory.add_argument(
+        '-d',
+        metavar=Metavar.str,
+        required=True,
+        choices=list(dict_url.keys()),
+        help=f"Name of the dataset. Choices are: {sorted(dict_url)}"
```

Comment:
same comment as above-- you don't need to add "Choices are...", if you remove metavar.